src/patternfly/components/Compass/compass.scss

```diff
@@ -68,13 +80,20 @@
 }
 
 .#{$compass}__nav {
+  --#{$compass}__panel--Padding: 0 var(--#{$compass}__nav--Padding); // assumes there are tabs pushing the height enough not to need block padding on the home/search buttons
```

Comment:
We should probably separate `__panel--Padding` into the 4 sides. Do you mind doing that in this PR? The only other place you'd need to update the use of `__panel--Padding` is the sidebar padding here - https://github.com/srambach/patternfly/blob/432fb30ea563b5f069002910420b37e57e263d85/src/patternfly/components/Compass/compass.scss#L98.